user/urls.py

```diff
@@ -1 +1,11 @@
-# write your code here
+from django.urls import path
+
+from user.views import CreateUserView, CreateTokenView, ManageUserView, OrderViewSet
+
+urlpatterns = [
+	path("register/", CreateUserView.as_view(), name="create"),
+    path("login/", CreateTokenView.as_view(), name="token"),
+    path("me/", ManageUserView.as_view(), name="manage"),
+    path("order/", OrderViewSet.as_view(), name="order"),
```

Comment:
The `OrderViewSet` is being used with `as_view()`, which is not typical for viewsets. Viewsets usually require a router to automatically generate the URL patterns for the different actions (list, create, retrieve, update, destroy). Consider using a `DefaultRouter` or `SimpleRouter` to register the `OrderViewSet`.